core/modules/file.py

```diff
@@ -239,6 +239,34 @@ class File(Tree):
 
 	blobCacheTime = 60 * 60 * 24  # Requests to file/download will be served with cache-control: public, max-age=blobCacheTime if set
 
+	def write(self, filename: str, content: Any, mimetype: str = "text/plain", width: int = 0, height: int = 0):
```

Comment:
Please use `None` as explained in [this comment](https://github.com/viur-framework/viur-core/pull/424#discussion_r856369056) and add the return type.

```suggestion
	def write(self, filename: str, content: Any, mimetype: str = "text/plain", width: int = None, height: int = None) -> db.Key:
```